component/sections/how_it_works/index.js

```diff
@@ -1,17 +1,27 @@
-import React from "react"
-import dynamic from "next/dynamic"
+import React from "react";
+import dynamic from "next/dynamic";
+
+const Variants = {
+  variant_a: dynamic(() => import("./variant_a")),
+  variant_b: dynamic(() => import("./variant_b")),
+  variant_c: dynamic(() => import("./variant_c")),
+  variant_d: dynamic(() => import("./variant_d")),
+  variant_d: dynamic(() => import("./variant_d")),
```

Comment:
Kindly change `line 9` or the 5th variant to `variant_e`.
With current implementation, when variant_e is selected on studio, no web preview is shown as highlighted in this [video](https://www.loom.com/share/201070128a9f46fd98c6bfbc4a77c8f8).

Reply:
Thanks yes!

Reply:
Fix in [`8090572` (#18)](https://github.com/webriq-pagebuilder/site-template-default/pull/18/commits/809057252ba29b2728a584b373b8cc1fa35b8486)

Reply:
Approved. Confirming fix.